joblib/hashing.py

```diff
@@ -30,7 +30,7 @@ class _ConsistentSet(object):
         whatever the order of its items.
     """
     def __init__(self, set_sequence):
-        self._sequence = sorted(set_sequence)
+        self._sequence = sorted(set_sequence, key=hash)
```

Comment:
My mistake this is the joblib hash function that is deterministic, not the Python hash function. Please ignore this comment.